kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/checks/types/CheckContext.kt

```diff
@@ -158,6 +158,28 @@ public open class CheckContext<out T : Event>(public val event: T, public val lo
     ): String =
         translations.translate(key, locale, bundleName = bundle, replacements = replacements)
 
+    /** Quick access to translate strings using this check context's [locale]. **/
+    public fun translate(
+        key: String,
+        replacements: Array<Any?> = arrayOf()
+    ): String =
+        translations.translate(key, locale, bundleName = defaultBundle, replacements = replacements)
+
+    /** Quick access to translate strings using this check context's [locale]. **/
+    public fun translate(
+        key: String,
+        replacements: Map<String, Any?>
+    ): String =
+        translations.translate(key, locale, bundleName = defaultBundle, replacements = replacements)
+
+    /** Quick access to translate strings using this check context's [locale]. **/
+    public fun translate(
+        key: String,
+        bundle: String?,
+        replacements: Map<String, Any?>
+    ): String =
+        translations.translate(key, locale, bundleName = bundle, replacements = replacements)
+
```

Comment:
It seems like you've created duplicate functions here - I feel like it'd make more sense to have functions that take a `bundle` argument that defaults to `defaultBundle` and not provide functions that take no bundle, right?

Reply:
In the case of the first function in this block (key, replacements), this overload was already present in most of the other implementations ([example](https://github.com/Kord-Extensions/kord-extensions/pull/189/files/a305e9849b8ad40b3ed3c05d2caaea47f738edbd#diff-bb460ba4201be9a2852ab776870b77075a32408802712513759383971ae39456R81-R84)), but it was missing for CheckContext.

Without this function, you cannot call it like this:

```kt
// Doesn't compile, required String?, found Array... 
translate("check.positionalParameters", arrayOf(user.mention, user.id))

// This compiles
translate("check.positionalParameters", replacements = arrayOf(user.mention, user.id))
```

So the same is the case for the new functions with `Map`, without that overload with no bundles parameter, it forces you to use a named parameter, which is pretty annoying, and since this overload already existed for pretty much all other implementations, I also added it to these new translate methods.

Reply:
:+1: